Sources/TA/Extensions/User+.swift

```diff
@@ -0,0 +1,24 @@
+extension User {
+    public func move(to: Point) {
+        location = to
+    }
+}
+
+extension User: PlayerController {
+    // MARK: Actionable
+    public func act(on action: String) {
+        switch action {
+        case "down":
+            location.y += 1
+        case "up":
+            location.y -= 1
```

Comment:
up is down? 🤔

Reply:
Ya lol weird since it is printed out as a 2d array. Must be inverted